recipes/bdwgc/all/patches/update-cmake-8_0_4.patch

```diff
@@ -337,11 +337,9 @@
 +  endif()
 +  include_directories(libatomic_ops/src)
```

Comment:
This doesn't seem like the correct way to add the Atomic_ops include directory using the provided Conan package. Are the following lines not sufficient?

Reply:
Yes, let's remove it. (Upstream has include_directories(libatomic_ops/src) but for a different usage)

Reply:
But I suggest to keep it in update-cmake-8_2_2.patch - the smaller patch is the better, it does not hurt anyway

Reply:
Done (for 8.0.4 and 8.0.6)

Reply:
Thanks!